tox.ini

```diff
@@ -68,6 +68,40 @@ commands =
 	 # file.  See pyproject.toml for legal <change type> values.
 	 python -m towncrier.check
 
+[testenv:draftnews]
+basepython=python3.6
+passenv = TAHOE_LAFS_* PIP_* SUBUNITREPORTER_* USERPROFILE HOMEDRIVE HOMEPATH
+# Get "certifi" to avoid bug #2913. Basically if a `setup_requires=...` causes
+# a package to be installed (with setuptools) then it'll fail on certain
+# platforms (travis's OX-X 10.12, Slackware 14.2) because PyPI's TLS
+# requirements (TLS >= 1.2) are incompatible with the old TLS clients
+# available to those systems.  Installing it ahead of time (with pip) avoids
+# this problem.
+whitelist_externals = mv
+deps =
+     certifi
+     towncrier
```

Comment:
You could almost do `deps = {[testenv]deps}` and tox will behave as though you put in the same value as is in the `deps` field of the `testenv` section - thus avoiding duplicating the actual dependencies....

except that `{[testenv]deps}` contains certifi and _subunitreporter_ instead of certifi and _towncrier_, oops.

Ugh.  I guess I'd probably just make this comment say "See the comment for deps in testenv" instead of duplicating all the words, then.